### DISCUSSION TITLE
Is the zipformer2 supported by ncnn ?

### DISCUSSION THREAD

**Discussion by joazoa** [k2-fsa/icefall#1375] [Q&A] [answered]
Can somebody tell me if the zipformer2 works with ncnn?

Thank you!

**Comment by csukuangfj** [comment]
No, not yet. All supported modes are listed in
https://k2-fsa.github.io/icefall/model-export/export-ncnn.html

**Comment by csukuangfj** [comment, marked as answer]
All models from icefall are supported by sherpa-onnx.
